Patch origin: in-house
Patch status: not Solaris-specific; suitable for upstream

Syntax error. =back is missing.

--- openca-ocspd-3.1.2/docs/ocspd.3.pod	Tue Nov 10 13:31:40 2015
+++ openca-ocspd-3.1.2/docs/ocspd.3.pod	Thu Dec 17 13:07:42 2015
@@ -114,4 +114,4 @@
 
 =cut
 
-
+=back
